lusidtools/cocoon/utilities.py

```diff
@@ -1354,8 +1355,26 @@ def create_scope_id(time_generator=None):
     # Multiply by 7 to get value to 100s of nano seconds
     timestamp = hex(int(test * 10000000.0))
     # Create the scope id by joining the hex representation with dashes every 4 characters
-    scope_id = "-".join(timestamp[i : i + 4] for i in range(2, len(timestamp), 4))
-    return scope_id
+    uid = "-".join(timestamp[i : i + 4] for i in range(2, len(timestamp), 4))
```

Comment:
The `load_from_data_frame` function doesnt like `-` and it converts them to `-` but thats an issue with `load_from_data_frame` - we should probably fix that function to allow `-` as LUSID permits `-`

Reply:
Um whats the difference between the two `-` above? I cant see in the formatting!

Reply:
Underscore (_) versus hyphen (-).
The function like underscore.

Reply:
Ah ok, I must have miss-read as a hyphen